pysyft/types/torch/v1/tensor.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package pysyft.types.torch.v1;
+
+message Tensor {
```

Comment:
Although the tests seem to pass around `torch.Tensor` a lot, we'll also need a serialized tensor schema for Tensorflow and maybe a generic Syft tensor schema too.